requirements.txt

```diff
@@ -2,3 +2,5 @@ pandas
 httplib2
 google-api-python-client
 oauth2client
+google-cloud
```

Comment:
You can also be more specific and only require the bigquery library with `google-cloud-bigquery`: https://github.com/GoogleCloudPlatform/google-cloud-python/tree/master/bigquery